contrib/kube-nodered/README.md

```diff
@@ -0,0 +1,324 @@
+# Kube-node-red(en)
+[![hackmd-github-sync-badge](https://hackmd.io/cocSOGQMR-qzo7DHdwgRsQ/badge)](https://hackmd.io/cocSOGQMR-qzo7DHdwgRsQ)
+
+Kube-node-red is aiming to integrate Kubeflow/Kubebeters with node-red, leveraging node-red's low-code modules, and using Kubeflow resources (e.g. Kubeflow pipeline, Kserve) to enhance its AI/ML ability.
+## Architecture
+![5A0ECFB3-D5AC-4A89-8AD5-14696A9E0449](https://github.com/NightLightTw/kubeflow-Node-RED/assets/78789817/7cce84cf-a4df-47a6-9992-9412bc70819b)
+## Demo
+[![demo](https://i.ytimg.com/vi/72tXYl6FcvU/hqdefault.jpg)](https://youtu.be/72tXYl6FcvU)
+
+## Reference
+https://github.com/NightLightTw/kubeflow-Node-RED/tree/main
+
+https://github.com/kubeflow/pipelines/tree/1.8.21/backend/api/python_http_client
+
+[Kubeflow-Node-red implementation Random Forest training flo](https://hackmd.io/@Nhi7So-lTz2m5R6pHyCLcA/Sk1eZFTbh)
+
+
+# Usage instruction
+## Prerequsite
+
+As this project focused on the node-red integration with Kubeflow, one running Kubeflow instance should be ready on a publicly available network.
+(If you need to provision your own Kubeflow instance, you could refer to our [mulitkf](https://github.com/footprintai/multikf) project to allocate one instance for developing.)
+
+## Build Container Image
+
+You can proceed to [examples](./examples/README.md) folder and run the `./build.sh` to build the image locally or just run the `./run.sh` which would download the base image from our public repository.
+
+## Running examples
+
+We organized some examples under [examples](./examples/README.md) folder, and make sensitive information pass via environment variables. Please refer the following example to launch an individual example:
+```
+# If you use Windows system, it is recommended to use WSL
+$ git clone https://github.com/NightLightTw/kubeflow-Node-RED.git
+
+# enter folder
+cd kubeflow-Node-RED
+
+cd examples && \
+KUBEFLOW_HOST=<your-kubeflow-instance-endpoint> \
+KUBEFLOW_USERNAME=<your-username-account> \
+KUBEFLOW_PASSWORD=<your-password> \
+./run.sh <example-index>
+```
+> **Info:** Here <example-index>please use 1.connect-kubeflow
+        
+## User interface
+
+1. then you can go to UI, check it out: http://127.0.0.1:1880/
+![](https://hackmd.io/_uploads/HkoPcs4B3.png)
+
+2. press “install dependency”button to install dependency items such as specific python libraries and wait for its completion
+![](https://hackmd.io/_uploads/SymtCs4Sn.png)
+
+3. switch different pipelines and press the time stamp button to trigger the flow process
+![](https://hackmd.io/_uploads/Byn7JnEBn.png)
+> **Info:** If the environment variable does not work, please fill in the account password directly in the python file
+## Only test python files to interact with kubeflow
+```
+# Open another terminal and check docker status
```

Comment:
We do not support lecacy and deprecated Docker within the Kubeflow installation. We only support modern OCI containers which run with the official Kubernetes best practices PSS restricted set. Please check whether you satisfy that https://kubernetes.io/docs/concepts/security/pod-security-standards/. I strongly reccomend something modern such as podman etc. for local installations